br-webui/test/test-query-screen.html

```diff
@@ -0,0 +1,67 @@
+<script>
+   var companionScreens = {
+        "root" : [],
+        "pi" : []
+    }
+
+    function queryScreens(user, callback) {
```

Comment:
are we avoiding Jquery on purpose? (I ask because I try to)
Because since it is already being used, it makes things A LOT simpler: 
http://youmightnotneedjquery.com/#json

Reply:
not including it in this example file makes sense, but it is in our headers for companion already